### ISSUE TITLE
Question about scale_color_manual and colors to points mapping

### ISSUE THREAD

**Issue by dputhier** [has2k1/plotnine#169]
Hi,
In R, the mapping between colors and points is achieved through a named vector in the scale_color_manual function. I tried several solutions in plotnine, using list, list of tuples, dict, structured array (...). I'm unable to achieve proper mapping. Could you help me with this simple example:

      import pandas as pd
      from plotnine import  *
      m = pd.DataFrame({'x':range(15), 'y':range(15), 'c': ['A']*5+['B']*5+['C']*5})
      # What should I write to get points 'A' in red, 'B' in violet... ?
      # Maybe the most natural for a beginner is a dict (imho) 
      ggplot(m, aes('x','y', color='c')) + geom_point(size=3) + scale_color_manual({'A':'red', 'B':'violet', 'C': 'blue'})

Thank you

**Comment by has2k1** [comment]
In plotnine `scale_color_manual` and other manual scales accept only lists, arrays or tuples. I'll look into supporting a dict to allow explicity mapping.

**Comment by rohan-shah** [comment]
This is a feature I'd really like to have, too.

**Comment by glassTransition** [comment]
I would love to use this feature, but I can't find details about how. Is there any example code or can someone create a small example? It would be MOST appreciated!

**Comment by dputhier** [comment]
There is already an example above. Here the same but with some comments:

	import pandas as pd
	from plotnine import  *

	# An example dataframe 
	m = pd.DataFrame({'x':range(15), 'y':range(15), 'c': ['A']*5+['B']*5+['C']*5})
	print(m)

	# Plot x and y with colors mapped from 
	# a dict
	ggplot(m, aes('x','y', color='c')) + geom_point(size=3) + scale_color_manual({'A':'red', 'B':'violet', 'C': 'blue'})

**Comment by glassTransition** [comment]
Thank you so much for the quick response. I just wasn't sure that the implementation followed the original feature request. I am new to Pandas and Matplotlib and have spent countless hours trying different things for my dataset. Once I found Plotnine I was very encouraged, but started yet another learning curve that I feared I didn't have the time to complete. I hope this will solve my last issue.

All the best and thank you for this project.
